src/psyclone/dynamo0p3.py

```diff
@@ -7061,7 +7929,11 @@ def __init__(self, call, parent_call):
                                              parent_call)
             idx += 1
             if dyn_argument.descriptor.stencil:
+                # Create a stencil object and store a reference to it in our
```

Comment:
define interface of parent docstring please.

Reply:
Done.